server/routers/viewer.tsx

```diff
@@ -1,24 +1,19 @@
+import { Prisma } from "@prisma/client";
 import { TRPCError } from "@trpc/server";
+import { z } from "zod";
 
-import { createRouter } from "../createRouter";
+import checkPremiumUsername from "@ee/lib/core/checkUsername";
 
-// routes only available to authenticated users
-export const viewerRouter = createRouter()
-  // check that user is authenticated
-  .middleware(({ ctx, next }) => {
-    const { user } = ctx;
-    if (!user) {
-      throw new TRPCError({ code: "UNAUTHORIZED" });
-    }
+import checkRegularUsername from "@lib/core/checkUsername";
```

Comment:
These two should probably be named distinctly different things to avoid confusion

Reply:
As in `"@ee/lib/core/checkUsername"` // `"@lib/core/checkUsername"`

Also, I'm not a big fan of `export default` -- I prefer named exports

Reply:
Agreed